app/labor/sidekiq/worker_retries_exhausted_reporter.rb

```diff
@@ -0,0 +1,10 @@
+module Sidekiq
+  class WorkerRetriesExhaustedReporter
+    def self.report_final_failure(job)
```

Comment:
I'm quite sure without the explanation you added in the PR someone (aka me) is going to ask what is this for in a few weeks :D I think it's worth adapting the PR description as a comment to this method

Reply:
I feel like this middleware's name is pretty self-explanatory. I am all for commenting on code when it is obscure but I feel like the class name and method are pretty straightforward.